### ISSUE TITLE
Add new Widgets to dashboard

### ISSUE THREAD

**Issue by vpetersson** [WoTTsecurity/api#819]
Here's a mock-up for the new extensions to the dashboard (Sketch file available if needed):

![dashboard-UI](https://user-images.githubusercontent.com/357664/77650698-c5406a80-6f63-11ea-9df4-094ab3cdbb84.png)

Here are the new metrics and how to calculate them.

## Weekly Streak

tl;dr: Number of weeks where five RAs have been completed

## Weekly Progress

tl;dr: Number of completed RAs during the current week

## Quarterly Target

tl;dr: Number of RAs completed this quarter (out of a target of 60)

## Average Time to Remediation 

Placeholder for now, but will be the hours it took to remediate issues. There is a minefield of edge cases here, so let's just leave in the placeholder here.
